package.json

```diff
@@ -163,7 +165,7 @@
     "rollup-plugin-svg": "^2.0.0",
     "rollup-plugin-terser": "^7.0.2",
     "sinon": "^7.2.2",
-    "typescript": "4.3.5",
+    "typescript": "4.4.3",
```

Comment:
Just because I think this is super cool: https://code.visualstudio.com/updates/v1_60#_inlay-hints-for-javascript-and-typescript